src/components/Upload.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import styles from './Upload.scss';
+
+const renderDropzone = field => (
```

Comment:
Validating input is an area where it's hard to do overkill, more checks are almost always better.

That ES lint doesn't complain now is a shotcomming in ES lint.

You can check the redux-form docs if they provide you we predefined proptypes.